test/uk/gov/hmrc/economiccrimelevyregistration/views/ViewUtilsSpec.scala

```diff
@@ -56,4 +59,34 @@ class ViewUtilsSpec extends SpecBase {
     }
   }
 
+  "formatDate" should {
```

Comment:
Is there any benefit in specifying a date that is unambiguous for these tests, e.g. 2007-12-31?

Reply:
I deliberately chose the date 3 December 2007 as it proves that leading zeros aren't displayed in the output. I think the ambiguity in the parsing of the string into a LocalDate isn't an issue as that's not really what we're testing, the expected output makes it clear enough I think.